server/store/adapters/rdbms/upgrade_fixes.go

```diff
@@ -242,6 +243,13 @@ func fix_2023_03_00_migrateComposePageMeta(ctx context.Context, s *Store) (err e
 	)
 }
 
+func fix_2024_03_00_migrateNamespacePageBlocks(ctx context.Context, s *Store) (err error) {
+	return addColumn(ctx, s,
+		"compose_namespace",
+		&dal.Attribute{Ident: "blocks", Type: &dal.TypeJSON{DefaultValue: "[{}]"}},
```

Comment:
Oof, I don't know how much I like you giving it an empty object; feels like this will cause more issues then it solves.
Why not just `[]`?